package.json

```diff
@@ -2,6 +2,10 @@
   "name": "@ui-kit-2022/monorepo",
   "version": "0.0.0",
   "private": true,
+  "engines": {
+    "npm": ">=8.0.0 <9.0.0",
```

Comment:
We might also want to specify Yarn version 3.x here, too.

Reply:
Thanks, Ian. It has been added.